core/conn/jdbc_type4/src/org/trafodion/jdbc/t4/TrafT4ResultSet.java

```diff
@@ -4299,12 +4299,10 @@ private BaseRow getCurrentRow() throws SQLException {
 			return insertRow_;
 		} else {
 			if (isBeforeFirst_) {
-				String cursorExpt = "The cursor is before the first row, therefore no data can be retrieved.";
-				throw HPT4Messages.createSQLException(connection_.props_, connection_.getLocale(), cursorExpt, null);
+				throw HPT4Messages.createSQLException(connection_.props_, connection_.getLocale(), "cursor_is_before_first_row", null);
```

Comment:
There was another pull request that changed HPT4Messages to T4Messages. I don't know the details behind it, but I'm wondering if a similar change needs to be made here?